packages/react/src/components/Combobox/Combobox.stories.tsx

```diff
@@ -16,7 +16,7 @@ import { Divider } from '../Divider';
 import { Combobox } from './index';
 
 export default {
-  title: 'Komponenter/Combobox',
+  title: 'Komponenter/Avviklet/Combobox',
```

Comment:
How about we just put in a separate category? Was it weird having a status folder on the same level as other components? My first thought was, whats this `Avviklet` component? 😂 

```suggestion
  title: 'Avviklet/Combobox',
```

Reply:
This was what I did initially: 
But we already have categories for things outside 'komponenter'. And having 'Avviklet' as one does not fit in.
If me move it out here it would then have to be 'Avviklede komponenter' ?
![image](https://github.com/user-attachments/assets/9704ace4-b0ff-49aa-ace9-f75cef13b295)

Reply:
But what if we have utilities that are deprecated? I think just `Avviklet` should be fine. Its short and descriptive :) 

You will either way click into it or see by the name and understand what it is.